app/views/index.erb

```diff
@@ -0,0 +1,48 @@
+<! DOCTYPE html>
```

Comment:
You can remove DOCTYPE, html tags and body. These are defined for you - in layout